nas_spec/components/schemas/Forex/ForexRate.yaml

```
@@ -0,0 +1,11 @@
+type: object
+properties:
+  exchange_rate:
+    description: The FX rate with Checkout.com margin applied. Up to 8 decimal places provided in response.
```

Suggestion to align with [brand guidelines](https://brand.checkout.com/document/1#/writing-for-cko/numbers/figures):

```suggestion
    description: The FX rate with Checkout.com margin applied. Up to eight decimal places provided in response.
```